sdk_container/src/third_party/coreos-overlay/dev-lang/rust/rust-1.79.0.ebuild

```diff
@@ -510,6 +511,26 @@ src_configure() {
 			linker = "${S}/cc.sh"
 			ar = "aarch64-cros-linux-gnu-ar"
 		EOF
+		cat <<- 'EOF' > "${S}/cc-riscv.sh"
```

Comment:
We need to make sense of a better approach here

Reply:
These two commands were also needed, as the rustc has a hardcoded value to the ar and compiler.

sudo ln -sf /usr/x86_64-pc-linux-gnu/riscv64-cros-linux-gnu/gcc-bin/13/riscv64-cros-linux-gnu-gcc /usr/bin/riscv64-sudo ln -sf /usr/libexec/gcc/riscv64-cros-linux-gnu/ar /usr/bin/riscv64-linux-gnu-ar